data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
 
 
 
+mission "Remnant: Tech Retrieval"
+	name "Remnant Human Tech Retrieval"
+	description "The Remnant are interested in learning what advances humanity has made over the past centuries. Bring them samples of advanced technology."
+	source "Viminal"
+	to offer
+		has "Remnant: Technology Available: offered"
+		random < 30
+	on offer
+		"remnant taely" ++
+		conversation
+			`As you enter the spaceport, you notice someone across the plaza appears to have been waiting for you. A tall, solidly built woman in short-sleeved coveralls stands up from where she was sitting at the information desk. With a few quick gestures at the person behind the counter, she heads towards you with a confident stride. As she approaches you note that the crowds of people moving around the plaza seem to staying out of her way as if she was someone important. In contrast, the tools at her belt and the wear marks on her clothing suggest someone who is more comfortable inside a ship access duct than behind a desk.`
```

Comment:
"A tall, solidly built woman in short-sleeved coveralls stands up from where she was sitting at the information desk. With a few quick gestures at the person behind the counter..." It's a little confusing where Taels is relative to the desk. Perhaps say, "where she was sitting NEAR the information desk.

Reply:
Changed `where she was sitting at the information desk` to `where she was sitting near the information desk`